spec/acceptance/mysql_spec.rb

```diff
@@ -0,0 +1,365 @@
+require 'acceptance_spec_helper'
+
+RSpec.describe 'MySQL sessions and MySQL modules' do
+  include_context 'wait_for_expect'
+
+  TESTS = {
+    mysql: {
+      target: {
+        session_module: "auxiliary/scanner/mysql/mysql_login",
+        type: 'MySQL',
+        platforms: [:linux, :osx, :windows],
+        datastore: {
+          global: {},
+          module: {
+            username: ENV.fetch('MYSQL_USERNAME', 'root'),
+            password: ENV.fetch('MYSQL_ROOT_PASSWORD', 'password'),
+            rhost: ENV.fetch('MYSQL_RHOST', '127.0.0.1'),
+            rport: ENV.fetch('MYSQL_RPORT', '3306'),
+          }
+        }
+      },
+      module_tests: [
+        {
+          name: "post/test/mysql",
+          platforms: [:linux, :osx, :windows],
+          targets: [:session],
+          skipped: false,
+        },
+        {
+          name: "auxiliary/scanner/mysql/mysql_hashdump",
+          platforms: [:linux, :osx, :windows],
+          targets: [:session, :rhost],
+          skipped: false,
+          lines: {
+            all: {
+              required: [
+                /Saving HashString as Loot/
+              ]
+            },
+          }
+        },
+        {
+          name: "auxiliary/scanner/mysql/mysql_version",
+          platforms: [:linux, :osx, :windows],
+          targets: [:session, :rhost],
+          skipped: false,
+          lines: {
+            all: {
+              required: [
+                /127.0.0.1:\d+ is running MySQL \d+.\d+.*/
```

Comment:
It could be a remote host

```suggestion
                /\d+\.\d+\.\d+\.\d+:\d+ is running MySQL \d+.\d+/
```